Makefile

```diff
@@ -1119,13 +1119,15 @@ pkg: $(PKG)
 
 .PHONY: corepack-update
 corepack-update:
-	rm -rf /tmp/node-corepack-clone
-	git clone 'https://github.com/nodejs/corepack.git' /tmp/node-corepack-clone
-	cd /tmp/node-corepack-clone && yarn pack
-	rm -rf deps/corepack && mkdir -p deps/corepack
-	cd deps/corepack && tar xf /tmp/node-corepack-clone/package.tgz --strip-components=1
+	mkdir -p /tmp/node-corepack
+	wget -q "$$(npm view corepack dist.tarball)" -O /tmp/node-corepack/package.tgz
+
+	rm -rf deps/corepack && mkdir deps/corepack
+	cd deps/corepack && tar xf /tmp/node-corepack/package.tgz --strip-components=1
 	chmod +x deps/corepack/shims/*
 
+	node deps/corepack/dist/corepack.js --version
+
```

Comment:
That doesn't seem to be necessary if we add it in the YAML file already.
```suggestion
```

Reply:
It was mostly just to confirm the update works fine (in case Corepack accidentally adds a runtime dependency), if someone was to run the command locally.